### DISCUSSION TITLE
blog/2023/02/16/what-happens-if-you-put-http-server-inside-postgres/

### DISCUSSION THREAD

**Discussion by giscus** [yrashk/yrashk.github.io#4] [General]
# blog/2023/02/16/what-happens-if-you-put-http-server-inside-postgres/

Writings on software development and work practices, entrepreneurship, tech in general.

https://yrashk.com/blog/2023/02/16/what-happens-if-you-put-http-server-inside-postgres/

**Comment by Jeevananthan-23** [comment]
Looking forward to work on future enhancements including HTTPS and HTTP/2( gRPC) and HTTP/3(Quic).

**Comment by unibum** [comment]
Very interesting article.. any chance you could put up (or add to this article) a basic architecture diagram to demonstrate how a cache (e.g. memcached), APIGW, webserver (since H2 [excellent choice or Axum/Salvo for openapi gen] is in Omnigres), pgbouncer, jwt generation? etc. would fit into the picture?
